src/main/java/org/prebid/server/bidder/HttpBidderRequester.java

```diff
@@ -132,19 +134,30 @@ public HttpBidderRequester(HttpClient httpClient,
 
         if (errorType == BidderError.Type.timeout) {
             if (bidder instanceof TimeoutBidder) {
-                final TimeoutBidder<T> timeoutBidder = (TimeoutBidder<T>) bidder;
-                final HttpRequest<Void> timeoutNotification = timeoutBidder.makeTimeoutNotification(httpRequest);
-                if (timeoutNotification != null) {
-                    httpClient.request(timeoutNotification.getMethod(), timeoutNotification.getUri(),
-                            timeoutNotification.getHeaders(), timeoutNotification.getBody(), NOTIFICATION_TIMEOUT_MS);
-                }
+                doTimeoutNotificationRequest(httpRequest, bidder);
             }
         }
 
         return Future.succeededFuture(
                 HttpCall.failure(httpRequest, BidderError.create(exception.getMessage(), errorType)));
     }
 
+    /**
+     *  doTimeoutNotificationRequest shoots on {@link TimeoutException} or {@link ConnectTimeoutException},
+     *  if Bidder supports Timeout notifications
+     */
+    private <T> void doTimeoutNotificationRequest(HttpRequest<T> httpRequest, Bidder<T> bidder) {
```

Comment:
please add   `if (bidder instanceof TimeoutBidder) {` to this method, or perform 
`  final TimeoutBidder<T> timeoutBidder = (TimeoutBidder<T>) bidder;` near instanceOf and change method parameter from Bidder to `TimeoutBidder`.